kyo-bench/src/main/scala/kyo/bench/Bench.scala

```diff
@@ -1,17 +1,11 @@
 package kyo.bench
 
-import cats.effect.IO
```

Comment:
Imports reorganized to avoid conflicts now that both kyo and cats-effect have an `IO` type.